x-pack/plugin/ilm/qa/with-security/src/test/java/org/elasticsearch/xpack/security/PermissionsIT.java

```diff
@@ -126,7 +133,46 @@ public void testCanViewExplainOnUnmanagedIndex() throws Exception {
         assertOK(client().performRequest(request));
     }
 
-    private void createNewSingletonPolicy(String policy, String phaseName, LifecycleAction action) throws IOException {
+    public void testILMPolicyWithRolloverActionWhereUserRoleIsLimitedByAliasOfIndex() throws IOException, InterruptedException {
```

Comment:
this test makes sense to me!

Since there is a lot of setup to get the user, index, and alias in the right state for the test, 
maybe some extra code comments could help people new to the code know what is being 
tested at a glance.